cloudify_agent/resources/pm/detach/detach.template

```diff
@@ -22,4 +22,5 @@
 --with-gate-keeper \
 --gate-keeper-bucket-size={{ max_workers }} \
 --with-logging-server \
---logging-server-logdir={{ workdir }}/logs
+--logging-server-logdir={{ workdir }}/logs \
+--heartbeat-interval=5
```

Comment:
`--heartbeat-interval` is not strictly required (we use the AMQP-level heartbeat as well) but it can be useful to force failover/reconnect in weird edge cases